ndsl/stencils/basic_operations.py

```diff
@@ -61,6 +61,28 @@ def adjust_divide_stencil(adjustment: FloatField, q_out: FloatField):
         q_out = q_out / adjustment
 
 
+def select_k(
```

Comment:
Can we get a `TODO` comment in there to refactor with `THIS_K` so that we don't need the mask and it's more elegant

Reply:
🫡